test/src/integration/integration_test.go

```diff
@@ -81,13 +81,18 @@ func TestEndToEndK8s(t *testing.T) {
 
 		scalarModules := []string{"kubernetes", "cassandra", "network"}
 
+
+
 		for _, m := range scalarModules {
 			terraformOptions := &terraform.Options{
 				TerraformDir: *terraformDir + *cloudProvider + "/" + m,
 				Vars:         map[string]interface{}{},
 				NoColor:      true,
 			}
-
+			if m == "kubernetes" {
+				terraform.RunTerraformCommand(t, terraformOptions, "init")
+				terraform.RunTerraformCommand(t, terraformOptions, "state", "rm", "module.kubernetes.kubernetes_config_map.aws_auth")
```

Comment:
Have you done gofmt?

Reply:
No I did not know about this.
Thank you I will take a look.